src/_base/docker/image/lighthouse/Dockerfile.twig

```
@@ -3,7 +3,10 @@ FROM yukinying/chrome-headless-browser:latest
 USER root
 
 RUN apt update
+RUN apt install -y jq
+RUN apt install -y bc
 RUN apt install -y npm
```

The image doesn't need to be the smallest in final size or layers due to not being pushed anywhere, but it would still be a small speed improvement to work out the dependencies for all three packages in one go:
```suggestion
RUN apt update \
 && apt install -y bc jq npm
```